tempesta_fw/cache.c

```diff
@@ -151,10 +151,12 @@ static const int hbh_hdrs[] = {
 	[TFW_HTTP_HDR_CONNECTION]	= 1,
 };
 
-static struct {
-	int	cpu;
-	TDB	*db;
-} c_nodes[MAX_NUMNODES] __read_mostly;
+static struct ca_node_t {
+	int		cpu[NR_CPUS];
+	unsigned int	cpu_idx;
+	unsigned int	nr_cpus;
+	TDB		*db;
+} ca_nodes[MAX_NUMNODES];
```

Comment:
Ok, agree about `__read_mostly`, but disagree about more readable `ca`.

One more point: we use typedefs instead of `struct` type definitions. So there should be `typedef struct { .. } CaNode` instead of `struct ca_node_t`.